lib/core/module.js

```diff
@@ -22,6 +22,7 @@ export class Module<RawOpt: { [string]: any }, Opt: { [string]: ModuleOption<Raw
 	exclude: Array<PageType | AppType | RegExp> = [];
 	shouldRun: () => boolean = () => true;
 	onToggle: (enabling: boolean) => void = () => {};
+	onSaveSettings: (savedSettings: any) => void = () => {}; // savedSettings will only contain options that were just changed
```

Comment:
How about renaming `savedSettings` to `changedSettings`, and dropping the comment?

Reply:
Well. Why didn't I think of that? Change coming up